doc/howto/network_ovn_setup.md

```diff
@@ -1,33 +1,124 @@
+   +------+---------+---------------------+----------------------------------------------+-----------+-----------+
+   ```
+
+## Set up a LXD cluster on OVN
+
 ```{youtube} https://www.youtube.com/watch?v=1M__Rm9iZb8
 ```
 
-This will create a standalone OVN network that is connected to the parent network lxdbr0 for outbound connectivity.
+Complete the following steps to set up a LXD cluster that uses an OVN network.
+The instructions are for a cluster of four servers, where three of the servers run the distributed database for OVN and the OVN controller, and the fourth server runs only the OVN controller.
+See the linked YouTube video for the complete tutorial.
 
-Install the OVN tools and configure the OVN integration bridge on the local server:
+1. Complete the following steps on three of the four machines:
 
-```
-sudo apt install ovn-host ovn-central
-sudo ovs-vsctl set open_vswitch . \
-  external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
-  external_ids:ovn-encap-type=geneve \
-  external_ids:ovn-encap-ip=127.0.0.1
-```
+   a. Install the OVN tools:
 
-Create an OVN network and an instance using it:
+        sudo apt install ovn-central ovn-host
 
-```
-lxc network set lxdbr0 ipv4.dhcp.ranges=... ipv4.ovn.ranges=... # Allocate IP range for OVN gateways.
-lxc network create ovntest --type=ovn network=lxdbr0
-lxc init ubuntu:22.04 c1
-lxc config device override c1 eth0 network=ovntest
-lxc start c1
-lxc ls
-+------+---------+---------------------+----------------------------------------------+-----------+-----------+
-| NAME |  STATE  |        IPV4         |                     IPV6                     |   TYPE    | SNAPSHOTS |
-+------+---------+---------------------+----------------------------------------------+-----------+-----------+
-| c1   | RUNNING | 10.254.118.2 (eth0) | fd42:887:cff3:5089:216:3eff:fef0:549f (eth0) | CONTAINER | 0         |
-+------+---------+---------------------+----------------------------------------------+-----------+-----------+
-```
+   b. Stop OVN on each machine:
+
+        systemctl stop ovn-central
+
+   c. Note down the IP address of the machine:
+
+        ip -4 a
+
+1. On the fourth machine, install only `ovn-host`:
+
+        sudo apt install ovn-host nano
+
+1. On the first three machines, configure OVN:
+
+   a. Open `/etc/defaul/ovn-central` for editing.
+
+   b. Paste in the following configuration (replace `<server_1>`, `<server_2>` and `<server_3>` with the IP addresses of the respective machines, and `<local>` with the IP address of the machine that you are on):
+
+        OVN_CTL_OPTS="--db-nb-addr=<server_1> --db-nb-create-insecure-remote=yes --db-sb-addr=<server_1> --db-sb-create-insecure-remote=yes --db-nb-cluster-local-addr=<local> --db-sb-cluster-local-addr=<local> --ovn-northd-nb-db=tcp:<server_1>:6641,tcp:<server_2>:6641,tcp:<server_3>:6641 --ovn-northd-sb-db=tcp:<server_1>:6642,tcp:<server_2>:6642,tcp:<server_3>:6642"
+
+   c. Start OVN:
+
+        systemctl start ovn-central
```

Comment:
Should we also explicitly mark the service as enabled so its clear we expect it to start on boot using:

```
systemctl enable ovn-central
```

And the equivalent for ovn-host